src/jit/thunk.cc

```diff
@@ -0,0 +1,58 @@
+#include "thunk.h"
+#include "function-builder.h"
+#include "thread.h"
+#include "src/cast.h"
+#include "src/interp/interp.h"
+
+namespace wabt {
+namespace jit {
+
+Result_t InterpThunk(ThreadInfo* th, Index ind) {
+  auto* env = th->thread->env();
+  auto* func = cast<interp::DefinedFunc>(env->GetFunc(ind));
+
+  th->pc = func->offset;
+  th->thread->set_pc(func->offset);
+  th->thread->call_stack_top_ = th->call_stack - th->thread->call_stack_.data();
```

Comment:
I'm confused. Does `ThreadInfo::call_stack` point to the current stack frame?

Reply:
`ThreadInfo::call_stack` points to the first _available_ call frame in `Thread::call_stack_` or to one frame beyond the end of the call stack (`ThreadInfo::call_stack_max`) if it's full. When this thunk is called, the JITted code has already saved its information in a call frame, so `ThreadInfo::call_stack` points one frame beyond where the JITted code saved its PC.